code/edge/build.gradle

```diff
@@ -74,37 +82,40 @@ spotless {
         endWithNewline()
         licenseHeaderFile "../../config/formatter/adobe.header.txt"
     }
-}
-
-afterEvaluate {
-    tasks.withType(Javadoc).tap {
-        configureEach {
-            source = android.sourceSets.main.java.srcDirs
-            classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-            classpath += files(android.libraryVariants.collect { variant ->
-                variant.javaCompileProvider.get().classpath.files
-            })
-            options {
-                source = "8"
-                links "https://developer.android.com/reference"
-            }
-        }
+    kotlin {
+        target "src/*/java/**/*.kt"
+        ktlint('0.47.0')
+        licenseHeaderFile "../../config/formatter/adobe.header.txt"
     }
 }
 
-tasks.register('javadocInternal', Javadoc) {
-    destinationDir = reporting.file("javadocInternal")
-    options.memberLevel = JavadocMemberLevel.PRIVATE
+tasks.register('javadocJar', Jar) {
+    dependsOn dokkaJavadoc
+    archiveClassifier.set('javadoc')
+    from dokkaJavadoc.outputDirectory
 }
 
-tasks.register('javadocPublic', Javadoc) {
-    destinationDir = reporting.file("javadocPublic")
-    options.memberLevel = JavadocMemberLevel.PUBLIC
```

Comment:
how is the javadocMemberLevel especially for the public lib managed now?

Reply:
With the current change we are only generating docs for public classes.